tools/distpartitioning/utils.py

```diff
@@ -190,61 +221,28 @@ def augment_edge_data(edge_data, part_ids, id_offset):
         array of part_ids indexed by global_nid
     """
     #add global_nids to the node_data
-    global_eids = np.arange(id_offset, id_offset + len(edge_data[constants.GLOBAL_TYPE_EID]), dtype=np.int64)
+    #global_eids = np.arange(id_offset, id_offset + len(edge_data[constants.GLOBAL_TYPE_EID]), dtype=np.int64)
+    #edge_data[constants.GLOBAL_EID] = global_eids
+    etype_offset = {}
+    offset = 0
+    for etype_name, tid_range in edge_tids.items(): 
+        assert int(tid_range[0][0]) == 0
+        assert len(tid_range) == world_size
+        etype_offset[etype_name] = offset + int(tid_range[0][0])
+        offset += int(tid_range[-1][1])
```

Comment:
similarly, this is to convert the per-type edge ID ranges to global edge ID ranges.

Reply:
why don't you create one function for this like you did for nodes?

Reply:
This function no longer exists, because it was never used. So removed this entire function.